.github/workflows/build-and-test.yaml

```diff
@@ -36,7 +36,18 @@ jobs:
           target-packages: ${{ steps.get-self-packages.outputs.self-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Check file existence
```

Comment:
BTW, what do you think about making this a composite action in `autoware-github-actions`?

Reply:
I have created a new workflow `file-check`. I will replace it after the new workflow is merged.
https://github.com/autowarefoundation/autoware-github-actions/pull/41

Reply:
Thank you, I've commented. (I'm sorry for a lot of comments. :bow: )